### PR TITLE
Add internal release notes that were previously missing.

### PR DESCRIPTION
**Pull request by angerson**
I wasn't sure about some of the Important/Other changes, so please double-check
that I haven't missed anything actually critical.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -13,15 +13,111 @@
...
+    * Add `tf.contrib.distributions.bijectors.Reshape`.
+    * Support shape inference (i.e., shapes containing -1) in the Reshape bijector.
+* Add `streaming_precision_recall_at_equal_thresholds,` a method for computing
+  streaming precision and recall with `O(num_thresholds + size of predictions)`
+  time and space complexity.
+* Change RunConfig default behavior to not set a random seed, making random
+  behavior independently random on distributed workers. We expect this to
+  generally improve training performance. Models that do rely on determinism
+  should set a random seed explicitly.
+* Replaced the implementation of tf.flags with absl.flags.
```
Comment (av8ramit) [inline RELEASE.md]: Maybe use ` around each term.

File: RELEASE.md [hunk 2]
```diff
@@ -13,15 +13,111 @@
...
+    stochastic
+  * BFCAllocator changed to reduce internal fragmentation
+  * Add `cross_entropy` and `kl_divergence` to `tf.distributions.Distribution`.
+  * Add `tf.nn.softmax_cross_entropy_with_logits_v2` which enables backprop
+    w.r.t. the labels.
+  * GPU back-end now uses ptxas to compile generated PTX.
+  * BufferAssignment's protocol buffer dump is now deterministic.
+  * Renaming SqueezeDims attribute to Axis in C++ API for Squeeze op.
+  * Changed embedding op to use parallel version of DynamicStitch.
+  * Sparse multidimensional feature columns support
```
Comment (av8ramit) [inline RELEASE.md]: Punctuation.

File: RELEASE.md [hunk 3]
```diff
@@ -13,15 +13,111 @@
...
+  * It is now possible to iterate over Tensors
+  * Allow SparseSegmentReduction ops to have missing segment IDs.
+  * Modify custom export strategy to account for multidimensional sparse float
+    splits.
+  * Conv2D, Conv2DBackpropInput, Conv2DBackpropFilter now supports arbitrary
+    dilations with GPU and cudnnv6 suppor
+  * Estimator supports Dataset: `input_fn` can return a Dataset instead of
+    Tensors.
+  * RevBlock: memory-efficient implementation of reversible residual layers
+  * Minor refactor: move stats files from stochastic to common, remove
```
Comment (av8ramit) [inline RELEASE.md]: Punctuation.

File: RELEASE.md [hunk 4]
```diff
@@ -13,15 +13,111 @@
...
+    layers.
+  * `tf.data` now supports `tf.SparseTensor` components in dataset elements.
+    will ignore threads that remain running when tearing down infrastructure
+    after successfully completing training, instead of throwing a RuntimeError.
+  * It is now possible to iterate over Tensors
+  * Allow SparseSegmentReduction ops to have missing segment IDs.
+  * Modify custom export strategy to account for multidimensional sparse float
+    splits.
+  * Conv2D, Conv2DBackpropInput, Conv2DBackpropFilter now supports arbitrary
+    dilations with GPU and cudnnv6 suppor
```
Comment (av8ramit) [inline RELEASE.md]: support?

File: RELEASE.md [hunk 5]
```diff
@@ -13,15 +13,111 @@
...
+  * Add a way to run ops using a step function to MonitoredSession.
+  * Add DenseFlipout probabilistic layer.
+  * Add `DenseFlipout` probabilistic layer.
+  * A new flag `ignore_live_threads` is available on train. If set to True, it
+  * Restandardize `DenseVariational` as simpler template for other probabilistic
+    layers.
+  * `tf.data` now supports `tf.SparseTensor` components in dataset elements.
+    will ignore threads that remain running when tearing down infrastructure
+    after successfully completing training, instead of throwing a RuntimeError.
+  * It is now possible to iterate over Tensors
```
Comment (av8ramit) [inline RELEASE.md]: Punctuation.

File: RELEASE.md [hunk 6]
```diff
@@ -13,15 +13,111 @@
...
+* Other:
+  * Add necessary shape util support for bfloat16.
+  * Add a way to run ops using a step function to MonitoredSession.
+  * Add DenseFlipout probabilistic layer.
+  * Add `DenseFlipout` probabilistic layer.
+  * A new flag `ignore_live_threads` is available on train. If set to True, it
+  * Restandardize `DenseVariational` as simpler template for other probabilistic
+    layers.
+  * `tf.data` now supports `tf.SparseTensor` components in dataset elements.
+    will ignore threads that remain running when tearing down infrastructure
```
Comment (av8ramit) [inline RELEASE.md]: Reformat this sentence or maybe just delete the second portion.

File: RELEASE.md [hunk 7]
```diff
@@ -13,15 +13,111 @@
...
+  * Fix bug by casting dtype of `log_det_jacobian` to match `log_prob` in TransformedDistribution.
+  * Fix a bug in `import_meta_graph`'s handling of partitioned variables when
+  * Ensure `tf.distributions.Multinomial` doesn't underflow in `log_prob`.
+    Before this change, all partitions of an integer variable were initialized
+    with the shape of the unpartitioned variable; after this change they are
+    initialized correctly.
+* Other:
+  * Add necessary shape util support for bfloat16.
+  * Add a way to run ops using a step function to MonitoredSession.
+  * Add DenseFlipout probabilistic layer.
```
Comment (av8ramit) [inline RELEASE.md]: Duplicate.

File: RELEASE.md [hunk 8]
```diff
@@ -13,15 +13,111 @@
...
+  * `tf.data` now supports `tf.SparseTensor` components in dataset elements.
+    will ignore threads that remain running when tearing down infrastructure
+    after successfully completing training, instead of throwing a RuntimeError.
+  * It is now possible to iterate over Tensors
+  * Allow SparseSegmentReduction ops to have missing segment IDs.
+  * Modify custom export strategy to account for multidimensional sparse float
+    splits.
+  * Conv2D, Conv2DBackpropInput, Conv2DBackpropFilter now supports arbitrary
+    dilations with GPU and cudnnv6 suppor
+  * Estimator supports Dataset: `input_fn` can return a Dataset instead of
```
Comment (av8ramit) [inline RELEASE.md]: should Dataset be around ` ?

File: RELEASE.md (hunk 7 above)
Comment (angersson) [reply RELEASE.md]: Removed, thanks!

**Comment by angersson** [comment]
I fixed @av8ramit's suggestions and did some additional cleanup.

**Review by gunan — APPROVED** [review]

**Pull request merged**